src/main/java/seedu/weme/logic/commands/EditCommand.java

```diff
@@ -146,12 +146,12 @@ public void setName(Name name) {
             return Optional.ofNullable(name);
         }
 
-        public void setAddress(Address address) {
-            this.address = address;
+        public void setDescription(Description description) {
+            this.description = description;
         }
 
-        public Optional<Address> getAddress() {
-            return Optional.ofNullable(address);
+        public Optional<Description> getDescription() {
```

Comment:
Are we using `Optional`?

Reply:
In this case is because edit command checks to see if the user has input anything. If the user hasn't they use the original value. This case is ok, need to think about how optional fields should be handled in our add command though.